src/components/MultiHopTrade/hooks/useGetTradeQuotes/useGetTradeQuotes.tsx

```diff
@@ -289,6 +289,12 @@ export const useGetTradeQuotes = () => {
     )
   }, [combinedQuoteMeta, isDebouncing, isFetchingInput])
 
+  const isAnySwapperFetching = useMemo(() => {
+    return (
+      isDebouncing || isFetchingInput || combinedQuoteMeta.some(quoteMeta => quoteMeta.isFetching)
+    )
+  }, [combinedQuoteMeta, isDebouncing, isFetchingInput])
```

Comment:
duplicate of `isQuoteRequestIncomplete`

Reply:
So it is! https://github.com/shapeshift/web/pull/6088/commits/4f31f3ed4a1e9b011786827dfe8b162e17b1c6f8